amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpSinkStage.scala

```diff
@@ -9,22 +9,27 @@ import akka.util.ByteString
 import com.rabbitmq.client.AMQP.BasicProperties
 import com.rabbitmq.client._
 
-final case class OutgoingMessage(bytes: ByteString, immediate: Boolean, mandatory: Boolean, props: Option[BasicProperties])
+final case class OutgoingMessage(bytes: ByteString,
+                                 immediate: Boolean,
+                                 mandatory: Boolean,
+                                 props: Option[BasicProperties])
```

Comment:
Ugh, I would personally prefer this as 
```
case class Name(
    field1: Whatever,
    field2: Whatever)
```
As described in the good ol' scala style guide (http://docs.scala-lang.org/style/declarations) if possible.

I see the `NamedQueueSourceSettings` looking exactly "right" below.